### DISCUSSION TITLE
Did Ollama support break? 

### DISCUSSION THREAD

**Discussion by Decedis** [zed-industries/zed#13618] [Feature Requests]
Does this mean that we can use other LLM's to generate code?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
